docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,292 @@
+
+For code examples, see `here <#tabular-checks>`__
+
+All of these checks appear in the `deepchecks interactive demo <https://checks-demo.deepchecks.com>`__, where you can
+insert corruption into the data and see the check at work.
+
+Computer Vision Data
+--------------------
+
+All of the computer vision checks use the :doc:`image and label properties</user-guide/vision-properties>` to estimate
+drift, as image data and labels are not simple one-dimensional variables.
+
+To detect `data <#data-drift>`__ or `concept drift <#concept-drift>`__, deepchecks offers the
+:doc:`Image Property Drift check </checks_gallery/vision/train_test_validation/plot_image_property_drift>` which uses
+`univariate measures <#detection-by-univariate-measure>`__ and the :doc:`Image Dataset Drift check</checks_gallery/vision/train_test_validation/plot_image_dataset_drift>`
+which uses a `domain classifier <#detection-by-domain-classifier>`__ in order to detect multivariate drift.
+
+For label drift, deepchecks offers the :doc:`Label Drift check </checks_gallery/vision/train_test_validation/plot_train_test_label_drift>`, which also uses univariate `statistical test <#detection-by-statistical-test>`__.
+
+In cases where the label is not available, we strongly recommend to also use the :doc:`Prediction Drift check</checks_gallery/vision/model_evaluation/plot_train_test_prediction_drift>`,
+which uses the same methods but on the model's predictions, and can detect possible changes in the distribution of the label.
+
+For code examples, see `here <#computer-vision-checks>`__
+
+
+Code Examples
+=============
+
+
+Tabular Checks
+--------------
+
+:doc:`TrainTestFeatureDrift </checks_gallery/tabular/train_test_validation/plot_train_test_feature_drift>`:
+
+.. code-block:: python
+
+    from deepchecks.tabular.checks import TrainTestFeatureDrift
+    check = TrainTestFeatureDrift()
+    result = check.run(train_dataset=train_dataset, test_dataset=test_dataset)
+
+:doc:`WholeDatasetDrift </checks_gallery/tabular/train_test_validation/plot_whole_dataset_drift>`:
+
+.. code-block:: python
+
+    from deepchecks.tabular.checks import WholeDatasetDrift
+    check = WholeDatasetDrift()
+    result = check.run(train_dataset=train_dataset, test_dataset=test_dataset)
+
+:doc:`TrainTestLabelDrift </checks_gallery/tabular/train_test_validation/plot_train_test_label_drift>`:
+
+.. code-block:: python
+
+    from deepchecks.tabular.checks import TrainTestLabelDrift
+    check = TrainTestLabelDrift()
+    result = check.run(train_dataset=train_dataset, test_dataset=test_dataset)
+
+:doc:`TrainTestPredictionDrift </checks_gallery/tabular/model_evaluation/plot_train_test_prediction_drift>`:
+
+.. code-block:: python
+
+    from deepchecks.tabular.checks import TrainTestPredictionDrift
+    check = TrainTestPredictionDrift()
+    result = check.run(train_dataset=train_dataset, test_dataset=test_dataset)
```

Comment:
Model needed.

Reply:
yes you are correct now stop unresolving